doc/planar.xml

```diff
@@ -0,0 +1,314 @@
+    <Ref Attr="SubdigraphHomeomorphicToK4"/>, and 
+    <Ref Attr="SubdigraphHomeomorphicToK23"/>.
+    <P/>
+
+    This method uses the reference implementation in
+    &edge-addition-planarity-suite; by John Boyer of the algorithms described
+    in <Cite Key="Boyer2006aa"/>.
+
+<Example><![CDATA[
+gap> D := Digraph([[3, 5, 10], [8, 9, 10], [1, 4], [3, 6], 
+> [1, 7, 11], [4, 7], [6, 8], [2, 7], [2, 11], [1, 2], [5, 9]]);
+<digraph with 11 vertices, 25 edges>
+gap> KuratowskiOuterPlanarSubdigraph(D);
+[ [ 3, 5, 10 ], [ 9, 8, 10 ], [ 4 ], [ 6 ], [ 11 ], [ 7 ], [ 8 ], 
+  [  ], [ 11 ], [  ], [  ] ]
+gap> D := Digraph([[2, 4, 7, 9, 10], [1, 3, 4, 6, 9, 10], [6, 10], 
+> [2, 5, 8, 9], [1, 2, 3, 4, 6, 7, 9, 10], [3, 4, 5, 7, 9, 10], 
+> [3, 4, 5, 6, 9, 10], [3, 4, 5, 7, 9], [2, 3, 5, 6, 7, 8], [3, 5]]);
+<digraph with 10 vertices, 50 edges>
+gap> IsOuterPlanarDigraph(D);
+false
+gap> KuratowskiOuterPlanarSubdigraph(D);
+[ [  ], [  ], [  ], [ 8, 9 ], [  ], [  ], [ 9, 4 ], [ 7, 9 ], [  ], 
+  [  ] ]
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="PlanarEmbedding">
+<ManSection>
+  <Attr Name="PlanarEmbedding" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is a planar digraph, then 
+    <C>PlanarEmbedding</C> returns the list of out-neighbours
+    of a subdigraph of <A>digraph</A> such that each vertex's
+    neighbors are given in clockwise order. If <A>digraph</A> is not planar,
+    then <K>fail</K> is returned. <P/>
+    
+    The directions and multiplicities of any edges in <A>digraph</A> are
+    ignored by <C>PlanarEmbedding</C>.
+    <P/>
+
+    See also 
+    <Ref Prop="IsPlanarDigraph"/>.
+    <P/>
+
+    This method uses the reference implementation in
+    &edge-addition-planarity-suite; by John Boyer of the algorithms described
+    in <Cite Key="Boyer2006aa"/>.
+
+<Example><![CDATA[
+gap> D := Digraph([[3, 5, 10], [8, 9, 10], [1, 4], [3, 6], 
+> [1, 7, 11], [4, 7], [6, 8], [2, 7], [2, 11], [1, 2], [5, 9]]);
+<digraph with 11 vertices, 25 edges>
+gap> PlanarEmbedding(D);
+[ [ 3, 10, 5 ], [ 10, 8, 9 ], [ 4 ], [ 6 ], [ 11, 7 ], [ 7 ], [ 8 ], 
+  [  ], [ 11 ], [  ], [  ] ]
+gap> D := Digraph([[2, 4, 7, 9, 10], [1, 3, 4, 6, 9, 10], [6, 10], 
+> [2, 5, 8, 9], [1, 2, 3, 4, 6, 7, 9, 10], [3, 4, 5, 7, 9, 10], 
+> [3, 4, 5, 6, 9, 10], [3, 4, 5, 7, 9], [2, 3, 5, 6, 7, 8], [3, 5]]);
+<digraph with 10 vertices, 50 edges>
+gap> PlanarEmbedding(D);
+fail
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="OuterPlanarEmbedding">
+<ManSection>
+  <Attr Name="OuterPlanarEmbedding" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is an outer planar digraph, then 
+    <C>OuterPlanarEmbedding</C> returns the list of out-neighbours
+    of a subdigraph of <A>digraph</A> such that each vertex's
+    neighbors are given in clockwise order. If <A>digraph</A> is not outer planar,
```

Comment:
Analogous comment here. As things stand, the documentation for this is essentially identical to `PlanarEmbedding`, but they're capturing different properties of a graph.